server/database/config/db_build.sql

```diff
@@ -0,0 +1,67 @@
+BEGIN;
+  DROP TABLE IF EXISTS employees,questions,employees_questions,forms,employee_form,form_questions
+  CASCADE;
+
+
+
+CREATE TABLE employees
+(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50) NOT NULL,
+  email VARCHAR  ,
+  image TEXT,
+  date date,
+  position VARCHAR(50),
+  role VARCHAR
+);
+
+CREATE TABLE questions
+(
+  id SERIAL PRIMARY KEY,
+  employee_id INT,
+  FOREIGN KEY (employee_id)  REFERENCES employees(id),
+  type VARCHAR NOT NULL,
+  result_rate VARCHAR ,
```

Comment:
I see the result-rate should be INT, but if you have a reason to choose VARCHAR let`s discuss, please!

Reply:
this data type has been changed .